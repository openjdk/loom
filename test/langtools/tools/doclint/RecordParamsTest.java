/*
 * @test /nodynamiccopyright/
 * @bug 8004832 8284994
 * @summary Add new doclint package
 * @modules jdk.javadoc/jdk.javadoc.internal.doclint
 * @build DocLintTester
 * @run main DocLintTester -Xmsgs:all -ref RecordParamsTest.out RecordParamsTest.java
 */

/**
 * Comment.
 * @param a aaa
 * @param a aaa
 * @param z zzz
 */
public record RecordParamsTest(int a, int b, int c) {  }
