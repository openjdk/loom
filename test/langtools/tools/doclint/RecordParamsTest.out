RecordParamsTest.java:13: warning: @param "a" has already been specified
 * @param a aaa
   ^
RecordParamsTest.java:14: error: @param name not found
 * @param z zzz
          ^
RecordParamsTest.java:16: warning: no @param for b
public record RecordParamsTest(int a, int b, int c) {  }
       ^
RecordParamsTest.java:16: warning: no @param for c
public record RecordParamsTest(int a, int b, int c) {  }
       ^
1 error
3 warnings
